src/main/java/net/openhft/chronicle/threads/Pauser.java

```diff
@@ -46,26 +51,33 @@ static boolean getSleepy() {
         return procs < MIN_PROCESSORS;
     }
 
+    /**
+     * Returns a {@link Pauser} that either yields, pauses, or does not wait at all, based on system capabilities.
+     * It selects the most appropriate pauser based on CPU availability and specified minimal busyness.
+     *
+     * @param minBusy the minimal busyness period in microseconds before yielding or pausing
+     * @return the most appropriate {@link Pauser}
+     */
     static Pauser yielding(int minBusy) {
         SleepyWarning.warnSleepy();
         return SLEEPY ? sleepy()
                 : BALANCED ? balanced()
-                : new TimeoutPauser(minBusy);
+                : new YieldingPauser(minBusy);
```

Comment:
I think we need to mention this in the issue as well. In the release notes it would look like just the toString has changed, but the change is broader than just the toString.

Reply:
This removes duplication between `YieldingPauser` and `TimeoutPauser`. The difference is that the only thing YeildingPauser didn't do has been added, and now those classes are the same.